x/auth/module.go

```diff
@@ -0,0 +1,64 @@
+package auth
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// name of this module
+const ModuleName = "auth"
```

Comment:
Why not `const Name = "auth"` -> `auth.Name`?

Reply:
You're going to need to elabourate

Reply:
I think @jackzampolin is taking this from the `auth` module user perspective as in he'd like to reference the module's name simply as `Name` instead of `ModuleName`. @jackzampolin correct me if I am wrong

Reply:
Oh yeah that makes sense